Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -135,7 +135,10 @@ def run
 
           if args.repology? && !Utils::Curl.curl_supports_tls13?
             begin
-              ensure_formula_installed!("curl", reason: "Repology queries") unless HOMEBREW_BREWED_CURL_PATH.exist?
+              unless HOMEBREW_BREWED_CURL_PATH.exist?
+                require "formula"
```

Comment:
OK with this one though it's handling formula objects above so will have to have been required earlier already.